libflux/src/semantic/flatbuffers/mod.rs

```diff
@@ -0,0 +1,672 @@
+//! This module defines methods for serializing and deserializing MonoTypes
```

Comment:
You mean to separate MonoType and PolyType serialization from semantic Node serialization?

Reply:
Yeah, in the same way that `semantic::nodes` and `semantic::types` are separate modules (although in that case, they are siblings)

Reply:
I'm fine with that. But just so I'm clear, you'd like to split up the serialization code because MonoTypes and PolyTypes will have deserialization code whereas the rest of the semantic nodes will not? Is this correct?

Reply:
Hmm, I guess it could go either way. I'll leave it up to you.  But I think this file is likely to be huge and nodes vs. types seems like a reasonable way to break it up.  And as you say, we probably won't need have deserialization in Rust for a while.

@fchikwekwe Any opinion on this?  You are currently working on the same file.